reactive-banana/src/Reactive/Banana/Prim/High/Combinators.hs

```diff
@@ -34,8 +34,15 @@ liftBuild = lift
     Interpretation
 ------------------------------------------------------------------------------}
 interpret :: (Event a -> Moment (Event b)) -> [Maybe a] -> IO [Maybe b]
-interpret f = Prim.interpret $ \pulse -> runReaderT (g pulse) undefined
-    where
+interpret f xs = do
+    -- This event network is unused, but it can't be undefined due to the seq in cacheAndSchedule
+    eventNetwork <- do
+      actuated <- newIORef False
+      size <- newIORef 0
+      s <- newMVar =<< Prim.emptyNetwork
+      pure EventNetwork{ actuated, size, s }
```

Comment:
```suggestion
    eventNetwork <- compile $ pure ()
```

I would like to re-use a top-level definition here. 🤔

Reply:
Oh, that's better 🙈